docs/reference/cluster/tasks.asciidoc

```diff
@@ -184,7 +184,7 @@ The new `description` field contains human readable text that identifies the
 particular request that the task is performing such as identifying the search
 request being performed by a search task like the example above. Other kinds of
 task have different descriptions, like <<docs-reindex,`_reindex`>> which
```

Comment:
May I suggest this too since it is really near by?
```suggestion
tasks have different descriptions, like <<docs-reindex,`_reindex`>> which
```

Reply:
yeah will change it